api/services/orders/order_handler.go

```diff
@@ -0,0 +1,70 @@
+package orders
+
+import (
+	"net/http"
+	"sort"
+
+	"github.com/gorilla/mux"
+	"github.com/sockify/sockify/middleware"
+	"github.com/sockify/sockify/types"
+	"github.com/sockify/sockify/utils"
+)
+
+type Handler struct {
+	orderStore types.OrderStore
+	adminStore types.AdminStore
+}
+
+func NewHandler(orderStore types.OrderStore, adminStore types.AdminStore) *Handler {
+	return &Handler{orderStore: orderStore, adminStore: adminStore}
+}
+
+func (h *Handler) RegisterRoutes(router *mux.Router) {
+	router.HandleFunc("/orders", middleware.WithJWTAuth(h.adminStore, h.handleGetOrders)).Methods(http.MethodGet)
+
+}
+
+// Get all orders, possibly filtered by status
+func (h *Handler) handleGetOrders(w http.ResponseWriter, r *http.Request) {
+	status := r.URL.Query().Get("status")
+
+	orders, err := h.orderStore.GetOrdersByStatus(status)
+	if err != nil {
+		utils.WriteError(w, http.StatusInternalServerError, err)
+		return
+	}
+
+	sortOrdersByDateDesc(orders)
```

Comment:
The sorting should be done in the SQL like this `... SORT BY created_at DESC`, so you can remove this helper.